src/lnd_compatibility.rs

```diff
@@ -41,13 +42,19 @@ impl Compatibility {
         let version = &version_info.version;
         let tags = version_info.build_tags;
 
-        let min_version = String::from("0.16.2-beta");
-
-        if version < &min_version {
-            error!(
-                "Version {} is less than minimum required version {}",
-                &version, &min_version
-            );
+        let min_version = 16.2;
+
+        let re = Regex::new(r"\d+\.\d+").unwrap();
```

Comment:
Ah right, using regex here - needs a squash with the previous commit.

Reply:
Also, can't we just use the numerical values returned by [version](https://lightning.engineering/api-docs/api/lnd/versioner/get-version#verrpcversion) to save us the hassle here? `app_major/minor/patch`?